### PR TITLE
SAAS-3533: Added Spec File For ELBV2 WAF Enabled Plugin

### PR DESCRIPTION
**Pull request by sadeed12345**

### CODE REVIEW

File: plugins/aws/elbv2/elbv2WafEnabled.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ module.exports = {
     recommended_action: '1. Enter the WAF service. 2. Enter Web ACLs and filter by the region the Application Load Balancer is in. 3. If no Web ACL is found, Create a new Web ACL in the region the ALB resides and in Resource type to associate with web ACL, select the Load Balancer. ',
     apis: ['ELBv2:describeLoadBalancers', 'WAFV2:listWebACLs', 'WAFRegional:listWebACLs', 'WAFV2:listResourcesForWebACL', 'WAFRegional:listResourcesForWebACL'],
     run: function(cache, settings, callback) {
+        // console.log(JSON.stringify(cache, null, 2));
```
Comment (AkhtarAmir) [inline plugins/aws/elbv2/elbv2WafEnabled.js:14]:
```suggestion
```

File: plugins/aws/elbv2/elbv2WafEnabled.js [hunk 2]
```diff
@@ -44,9 +44,10 @@ module.exports = {
 
             combinedACLS.forEach(webACL => {
                 if (webACL.WebACLId) {
+                    // console.log(JSON.stringify(cache.wafregional, null, 2));
```
Comment (AkhtarAmir) [inline plugins/aws/elbv2/elbv2WafEnabled.js:47]:
```suggestion
```

File: plugins/aws/elbv2/elbv2WafEnabled.js [hunk 3]
```diff
@@ -46,7 +46,6 @@ module.exports = {
                 if (webACL.WebACLId) {
                     let listResources = helpers.addSource(cache, source,
                         ['wafregional', 'listResourcesForWebACL', region, webACL.WebACLId]);
```
Comment (AkhtarAmir) [inline plugins/aws/elbv2/elbv2WafEnabled.js:48]:
```suggestion
                        ['wafregional', 'listResourcesForWebACL', region, webACL.WebACLId]);

```

File: plugins/aws/elbv2/elbv2WafEnabled.spec.js [hunk 4]
```diff
@@ -0,0 +1,218 @@
+var expect = require('chai').expect;
+const elbv2WafEnabled = require('./elbv2WafEnabled');
+
+const describeLoadBalancers = [
+    {
+        "LoadBalancerArn": "arn:aws:elasticloadbalancing:us-east-1:101363889637:loadbalancer/app/sad-elb/0c48be96f812e564",
```
Comment (AkhtarAmir) [inline plugins/aws/elbv2/elbv2WafEnabled.spec.js:6]: Can you remove account id from this?

**Review by AkhtarAmir — CHANGES_REQUESTED** [review]

File: plugins/aws/elbv2/elbv2WafEnabled.spec.js (hunk 4 above)
Comment (sadeed12345) [inline plugins/aws/elbv2/elbv2WafEnabled.spec.js:6]:
```suggestion
        "LoadBalancerArn": "arn:aws:elasticloadbalancing:us-east-1:000111222333:loadbalancer/app/sad-elb/0c48be96f812e564",
```

**Review by AkhtarAmir — APPROVED** [review]

**Pull request merged**